package.json

```diff
@@ -43,22 +43,22 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-promise": "^6.0.0",
     "husky": "^8.0.1",
-    "lint-staged": "^12.3.5",
     "jest": "^29.7.0",
-    "jest-environment-jsdom": "^28.1.1",
+    "jest-environment-jsdom": "^29.7.0",
+    "lint-staged": "^12.3.5",
     "prettier": "^2.7.1",
     "terser-webpack-plugin": "^5.3.3",
     "testdouble": "^3.16.6",
     "ts-jest": "^29.1.1",
     "ts-loader": "^9.3.1",
+    "ts-node": "^10.9.2",
     "typescript": "^4.7.4",
     "webpack": "^5.73.0",
     "webpack-cli": "^4.10.0",
     "xhr-mock": "^2.5.1"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "2.0.0",
-    "axios": "^1.6.0",
+    "@eppo/js-client-sdk-common": "file:../js-client-sdk-common",
```

Comment:
We will publish js-client-sdk-common before merging this PR, and replace this with its latest release version.